src/components/Badge/Badge.less

```diff
@@ -0,0 +1,19 @@
+@badgeBackground: #ccc;
+@badgeColor: #fff;
+@badgeFontWeight: bold;
```

Comment:
Didn't we come to terms on the `@Component-style` format for naming these variables? e.g. `@Badge-background`, `@Badge-font-weight`